hw4_hugging/imdb_emo_and_summary.py

```
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+get_ipython().system('pip install transformers datasets sentencepiece accelerate torch')
```

Ruff (0.13.3)

Undefined name `get_ipython`

(F821)